DOCKER.md

```diff
@@ -138,20 +150,40 @@ Replace `${HOME}/bundles` with the actual path to your bundles directory if need
 
```

Comment:
I think all of the ## Usage Examples section should probably be refocused to use https://github.com/modelcontextprotocol/inspector

You can build the container, launch the Inspector, and then tell it to use ./scripts/run.sh as the launch command to properly test the container with Inspector. Then people can use the correct workflow, have a UI to navigate, etc. Let's just replace all of this section with Inspector.